### PR TITLE
SEO-174767-WPF-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the missing H1.

Regards,
Beryl.

### CODE REVIEW

File: wpf/Classic/Chart/How-to/Change-the-position-of-the-Chart-Legend.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ control: Chart (Classic)
 documentation: ug
 ---
 
-## Change the position of the Chart Legend
+# Change the position of the Chart Legend
```
Comment (Yvone-Atieno) [inline wpf/Classic/Chart/How-to/Change-the-position-of-the-Chart-Legend.md:10]: H1 must contain platform name, include platform name

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: wpf/Classic/Chart/How-to/Change-the-position-of-the-Chart-Legend.md (hunk 1 above)
Comment (berylouma) [reply wpf/Classic/Chart/How-to/Change-the-position-of-the-Chart-Legend.md]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**